core/pom.xml

```diff
@@ -408,6 +372,20 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-shade-plugin</artifactId>
+        <configuration>
+          <shadedArtifactAttached>false</shadedArtifactAttached>
+          <artifactSet>
+            <includes>
+              <include>com.google.guava:guava</include>
+              <include>org.apache.spark:spark-network-common_${scala.binary.version}</include>
+              <include>org.apache.spark:spark-network-shuffle_${scala.binary.version}</include>
```

Comment:
Yes. From the first comment:

> For this reason, also, the network/ dependencies are shaded into the spark-core artifact too, so that the raw Guava dependency doesn't leak.